docs/response.rst

```diff
@@ -52,3 +52,29 @@ If a view function returns a list of objects, the :class:`Schema
         @blp.response(200, PetSchema(many=True))
         def get(self, args):
             return Pet.get()
+
+.. note:: If a view function returns a :class:`werkzeug.BaseResponse`, that
+   response object is returned unchanged: it is not dumped by the schema and
+   the status code is not applied.
+
+.. note:: If a view function returns a tuple containing a status code, this
+   status code is used in place of the one specified as ``response`` parameter.
+   Doing this is generally a bad idea because the response status code won't
+   match the code in the API documentation.
+
+Documenting Alternative Responses
+=================================
+
+The :meth:`Blueprint.response <Blueprint.response>` decorator is meant to
+generate and document the response corresponding to the "normal" flow of the
+function. There can be alternative flows, if the function raises an exception,
+which results in a ``HTTPException``, or if it returns a ``Response`` object
+which is returned as is.
+
+Those alternative responses can be documented using the
+:meth:`Blueprint.response <Blueprint.alt_response>` decorator. Its signature is
```

Comment:
I think the first part should be `Blueprint.alt_response`. Maybe the second can be `Blueprint.response`.